integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/DatadogInterceptor.kt

```diff
@@ -224,7 +224,8 @@ internal constructor(
             (sdkCore?.internalLogger ?: InternalLogger.UNBOUND).log(
                 InternalLogger.Level.WARN,
                 InternalLogger.Target.USER,
-                WARN_RUM_DISABLED.format(Locale.US, prefix)
+                WARN_RUM_DISABLED.format(Locale.US, prefix),
+                onlyOnce = true
```

Comment:
here maybe we should use `INFO` and allow multiple logs? wdyt? the idea is that `DatadogInterceptor` can start working before SDK is initialized and we need to let user know that __some__ calls will not be tracked.

Reply:
That makes sense indeed.